src/sumologic-span-processor/index.spec.ts

```diff
@@ -10,8 +10,14 @@ import { SumoLogicSpanProcessor } from './index';
 delete window.location;
 window.location = new URL('https://www.unit-test-example.com');
 
-jest.useFakeTimers();
-jest.spyOn(global, 'setTimeout');
+beforeAll(() => {
+  jest.useFakeTimers();
+  jest.spyOn(global, 'setTimeout');
+});
+
+afterAll(() => {
+  jest.useRealTimers();
```

Comment:
Why do you need to clean up after all the tests have run? Don't we finish the process then?